src/backend/RenderPass.cpp

```diff
@@ -51,6 +52,28 @@ namespace backend {
         return other == this;
     }
 
+    void RenderPassBase::ComputeAttachmentsFirstSubpasses(std::vector<uint32_t>* attachmentsFirstSubpasses) const {
```

Comment:
Ah I see it is done per-backend. I think it is a generally useful information and could be stored at the RenderPassBase level.